Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -92,8 +92,18 @@ the fifth argument before the third argument. This is used in `to_dual` to swap
 `≤`, `<` and `⟶`. It is also used in `to_additive` to translate from `^` to `•`.
 -/
 syntax toAdditiveReorderOption := &"reorder" " := " (num+),+
+/--
+`dont_translate := ...` takes a list of type variables that should not be considered
+for translation. For example in
```

Comment:
```suggestion
`dont_translate := ...` takes a list of type variables (separated by spaces) that should not be 
considered for translation. For example in
```

Is it worth adding a test / example where we apply `dont_translate` to multiple variables?